apps/files/src/components/FileList.vue

```diff
@@ -6,9 +6,9 @@
           <thead>
             <oc-table-row>
               <oc-table-cell shrink type="head">
-                <oc-checkbox class="uk-margin-small-left" id="filelist-check-all" @click.native="toggleAll" :value="selectedAll" />
+                <oc-checkbox class="uk-margin-small-left" id="filelist-check-all" labelVisuallyHidden="true" label="Select all items" @click.native="toggleAll" :value="selectedAll" />
               </oc-table-cell>
-              <oc-table-cell shrink type="head" v-if="!publicPage()" />
+              <oc-table-cell shrink type="head" v-if="!publicPage()"><span class="oc-visually-hidden">Favorites</span></oc-table-cell>
```

Comment:
Translate pls

Reply:
Done

Reply:
I still see here only string "Favorites" 😟

Reply:
Sorry, missed that instance. Fixed (for real)